plugins/rts54hub/fu-rts54hub-rtd21xx-device.c

```diff
@@ -193,14 +228,41 @@ fu_rts54hub_rtd21xx_device_i2c_read (FuRts54hubRtd21xxDevice *self,
 	return TRUE;
 }
 
+static void
+fu_rts54hub_rtd21xx_device_exit_background(FuRts54hubRtd21xxDevice *self, GError **error)
+{
+
+	guint8 buf[] = { ISP_BG_CMD_FW_UPDATE_EXIT };
+
+	if (!fu_rts54hub_rtd21xx_device_i2c_write (self,
+						   UC_ISP_SLAVE_ADDR,
+						   UC_BACKGROUND_OPCODE,
+						   buf, sizeof(buf),
+						   error))
+		g_prefix_error (error, "failed to ISP_BG_CMD_FW_UPDATE_EXIT: ");
+}
+
+static void
```

Comment:
if this is a void function, why are you passing error?  It seems like it's just going to cause double use of error variable.  I would suggest make this boolean and the use local error everywhere or make it void and don't have an error variable.